### ISSUE TITLE
Update `react-native-webrtc`

### ISSUE THREAD

**Issue by licaon-kter** [balzack/databag#70]
......to fix [CVE-2023-5217: Heap buffer overflow in vp8 encoding in libvpx](https://www.openwall.com/lists/oss-security/2023/09/28/5)

Wait on upstream: https://github.com/react-native-webrtc/react-native-webrtc/issues/1480

**Comment by balzack** [comment]
thanks for the heads up

**Comment by balzack** [comment]
I'm about to publish a release with the updated react-native-webrtc, but I am seeing this error in my local build:

2024-03-12 19:38:34,445 ERROR: Unused scanignore path: app/mobile/node_modules/react-native-webrtc
2024-03-12 19:38:34,445 ERROR: Could not build app com.databag: Can't build due to 1 error while scanning

The build succeeds without that line, but I don't understand why that line is no longer needed. Should I submit a PR to remove that scanignore?

**Comment by licaon-kter** [comment]
Do that, will look at the CI output then.

**Comment by balzack** [comment]
Should I remove the scanignore on the last built version in the yml ...or publish a new release, allow the build to fail, and do the PR on that failed version yml?

**Comment by licaon-kter** [comment]
Don't edit an already built version.

**Comment by balzack** [comment]
I opened a MR with a new build block. I apologize if I missed something obvious. Thank you for your help.

**Comment by balzack** [comment]
@licaon-kter is there any concern with me adding http support:

      android:usesCleartextTraffic="true"

Several people have requested that I support IP based addressing for communication over their LAN. The above line would need to be added to the AndroidManifest.xml

**Comment by licaon-kter** [comment]
It's not, but it's discouraged... :(

**Comment by balzack** [comment]
Sounds good. I will leave it off and inform the users that it's not supported.
If they really really want it, I can make an APK available for them.
Thanks for keeping fdroid awesome!

**Comment by licaon-kter** [comment]
I mean, self-signed does not work? That's half way there, not cleartext but no need for CAs and etc.

**Comment by balzack** [comment]
I think that might work, but I don't imagine it's easy to load the self-signed cert into the phone OS to establish trust with that server.

**Comment by balzack** [comment]
@licaon-kter is it okay to publish only the arm64-v8 arch?
I am trying to update the app in f-droid, but getting an error on the non arm64-v8 builds.

If not, have you seen this error before: "16 is less than minimum platform for arm (21)"
https://gitlab.com/rolandosborne/fdroiddata/-/jobs/11381664013

**Comment by licaon-kter** [comment]
Update line 3954 and the others like it to handle target 33 too, I guess
